pkg/utils/kubeutils/metadata.go

```diff
@@ -39,7 +39,6 @@ func ToKubeMetaMaintainNamespace(meta *core.Metadata) metav1.ObjectMeta {
 	return metav1.ObjectMeta{
 		Name:            meta.Name,
 		Namespace:       meta.Namespace,
-		ClusterName:     meta.Cluster,
```

Comment:
Why do we do this?

Reply:
I believe this has to deal with upgrading `k8s.io/apimachinery` as it is no longer an available property. [This might be of help](https://github.com/kubernetes/apimachinery/blob/fdfff894ab1ea5568846d99cddbca53d876aef77/pkg/apis/meta/v1/types.go#L258).